src/main/java/com/example/examplemod/Config.java

```diff
@@ -43,21 +44,24 @@ public class Config
     public static String magicNumberIntroduction;
     public static Set<Item> items;
 
-    private static boolean validateItemName(final Object obj)
-    {
-        return obj instanceof String itemName && BuiltInRegistries.ITEM.containsKey(ResourceLocation.parse(itemName));
+    private static boolean validateItemName(final Object obj) {
+        return obj instanceof String itemName
+                && BuiltInRegistries.ITEM.containsKey(ResourceLocation.parse(itemName));
     }
 
     @SubscribeEvent
-    static void onLoad(final ModConfigEvent event)
-    {
+    static void onLoad(final ModConfigEvent event) {
         logDirtBlock = LOG_DIRT_BLOCK.get();
         magicNumber = MAGIC_NUMBER.get();
         magicNumberIntroduction = MAGIC_NUMBER_INTRODUCTION.get();
 
         // convert the list of strings into a set of items
-        items = ITEM_STRINGS.get().stream()
-                .map(itemName -> BuiltInRegistries.ITEM.get(ResourceLocation.parse(itemName)))
-                .collect(Collectors.toSet());
+        items =
+                ITEM_STRINGS.get().stream()
+                        .map(
+                                itemName ->
+                                        BuiltInRegistries.ITEM.get(
+                                                ResourceLocation.parse(itemName)))
+                        .collect(Collectors.toSet());
```

Comment:
The close parenthesis are strange formatting choices:

```suggestion
                ITEM_STRINGS.get().stream()
                        .map(
                                itemName ->
                                        BuiltInRegistries.ITEM.get(
                                                ResourceLocation.parse(itemName)
                                        )
                        )
                        .collect(Collectors.toSet());
```
I'd expect something more like this.